rma_account/wizards/rma_refund.py

```diff
@@ -90,8 +105,16 @@ def compute_refund(self):
             return new_refund
 
     def invoice_refund(self):
-        rma_line_ids = self.env["rma.order.line"].browse(self.env.context["active_ids"])
-        for line in rma_line_ids:
+        rma_line_obj = self.env["rma.order.line"]
+        rma_obj = self.env["rma.order"]
+        active_ids = self.env.context.get("active_ids") or []
+        active_model = self.env.context.get("active_model")
+        if active_model == "rma.order":
+            rma = rma_obj.browse(active_ids)
+            lines = rma.rma_line_ids
+        else:
+            lines = rma_line_obj.browse(active_ids)
+        for line in lines:
```

Comment:
I think this part does not make sense. We check all lines from rma group (or selected rma lines from ui via active_ids) are approved... But then, the refund is created on the base of the wizard item_ids, which may not be related to the checked lines...
Example : select 2 rma lines first one is not approved, second one is ok.
Click on action => create refund. Delete the first line because you changed your mind => click on invoice_refund. It will block because the first line is not approved... But this line won't even be in the refund since it has been deleted from the wizard.

I'd just check the rma lines doing something like `lines = self.item_ids.line_id` instead. It workfs for both case (rma lines/rma group). Does it make sense to you?